forge/routes/api/team.js

```diff
@@ -88,10 +92,16 @@ module.exports = async function (app) {
         }
     })
 
-    app.get('/:teamId/projects', async (request, reply) => {
+    app.get('/:teamId/projects', { config: { allowAnonymous: true } }, async (request, reply) => {
```

Comment:
`allowAnonymous` is too open. That would allow anyone to query the list of projects without any authentication.

We already apply `verifyTokenOrSession` to all api routes, so the project token should be able to access this end point without any special handling. But having realised that, I see that is also too wide open.

I am going to merge this as-is, but will followup with a PR to improve the auth handling more generally and tighten things up for Token access.

Reply:
Nick, I am sure you realise this but even though i used the name `allowAnonymous` the token must be present otherwise this endpoint returns `Unauthorized` ...

![image](https://user-images.githubusercontent.com/44235289/181788506-b6bf409a-324c-4bf9-9bde-36f3b4da8cd3.png)

... due to existing pathways